models/dtl/wrt/pop_table.py

```diff
@@ -63,3 +63,44 @@ def app_pop(a, b):
         cur.close()
     con.commit()
     cur.close()
+
+# TODO: this loop is clunky af
+
+def census_loop(table_name):    
+    x = st_fips_get()
+    new_header = x.iloc[0]
+    x = x[1:]
+    x.columns = new_header
+    x.drop_duplicates(subset=['state'], inplace=True)
+    dup_x = x
```

Comment:
Part of the clunky-ness is that `st_fips_get` already returns a dataframe; the fact that the dataframe still needs massaging to be useful means that there more work to do in `st_fips_get`